### ISSUE TITLE
Female Focused Symptoms/ Conditions Updating 3

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7513]
**Preferred term label:**
Abnormal Vaginal Discharge

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pubmed ID: 23943752

**Parent term (use hpo.jax.org/app)**
unknown

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
n/a

**Comment by pnrobinson** [comment]
Added with definition:
Anomalous amount, odor, or consistency of the fluid or mucus that comes from the vagina
